src/plugins/data/common/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -319,7 +352,7 @@ export class IndexPattern implements IIndexPattern {
       title: this.title,
       timeFieldName: this.timeFieldName,
       sourceFilters: this.sourceFilters,
-      fields: this.fields.toSpec(),
+      fields: this.fields.toSpec({ getFormatterForField: this.getFormatterForField.bind(this) }),
```

Comment:
the `getFormatterForField` function https://github.com/elastic/kibana/pull/75185/files#diff-0b63d2b724ba854c7057cfe7ac846028R488 can then change to:

```
this.fieldFormats.getInstance(field.format.id, field.format.params)
```

Reply:
the link isn't opening to a line of code so I'm not sure I understand.

Reply:
i was trying to link to the getFormatterForField method on the index pattern class.